packages/kbn-unified-data-table/src/components/data_table.tsx

```diff
@@ -645,7 +645,7 @@ export const UnifiedDataTable = ({
     () =>
       onFieldEdited
         ? (fieldName: string) => {
-            closeFieldEditor.current = services.dataViewFieldEditor.openEditor({
+            closeFieldEditor.current = services?.dataViewFieldEditor?.openEditor({
```

Comment:
Since it's inside `useMemo` it would be nice to change line 648 to 

```
onFieldEdited && services?.dataViewFieldEditor
  ? ...
```

so `editField` becomes `undefined` when `dataViewFieldEditor` was not provided.

Reply:
Sure done